src/agentscope/agents/agent.py

```diff
@@ -64,7 +64,7 @@ def __call__(cls, *args: tuple, **kwargs: dict) -> Any:
                     ),
                     max_timeout_seconds=to_dist.pop(  # type: ignore[arg-type]
                         "max_timeout_seconds",
-                        1800,
+                        7200,
```

Comment:
Why we need to set the timout as 7200, which maybe too large

Reply:
To ensure the user's right code can be run successfully without any additional parameters.